pyproject.toml

```
@@ -79,14 +79,15 @@ classifiers = [
   'Programming Language :: Python :: 3.11',
   'Programming Language :: Python :: 3.12',
   'Programming Language :: Python :: 3.13',
+  'Programming Language :: Python :: 3.14',
   'Programming Language :: Python :: Implementation :: CPython',
   'Topic :: Scientific/Engineering :: Atmospheric Science',
   'Topic :: Scientific/Engineering :: Mathematics',
   'Topic :: Scientific/Engineering :: Physics'
 ]
 dependencies = [
   'attrs>=21.3',
-  'black>=24.10',
+  "black>=25.11",
```

I would expect the pre-commit toml formatter to catch this...
```suggestion
  'black>=25.11',
```